docs-src/docs.md

```diff
@@ -0,0 +1,422 @@
+
+![image alt text](images/image_5.png)
+
+See [GitHub Deploy Keys Documentation](https://developer.github.com/guides/managing-deploy-keys/#deploy-keys)
+
+### Run image
+Positive
+: Note: remove any previous currency-v1 deployments from the Apigee org in order to demonstrate proxies created successfully with correct revision numbers.
+
+All parameters required to run Jenkins and Maven are supplied using environment variables. These variables are set directly to the docker container with the docker run command.
+
+There is a script provided to make this process easier which can be found [here](https://github.com/flogei/ci-demo-walkthrough/blob/master/docs/setup/run-image.sh). Modify this script to provide values specific to your setup and execute the script to run the image.
+
+Please note that setup/run-image.sh script is in source control. Copy this file to setup/.run-image.sh before making any changes to prevent sensitive data committed to this repo. Root .gitignore file contains a directive to ignore any changes to setup/.run-image.sh.
+
+Go to the cloned folder
+
+```
+cp ./setup/run-image.sh ./setup/.run-image.sh
+```
+
+Modify the file and plug in your environment data:
+
+```
+vim ./setup/.run-image.sh
+```
+
+
+You get that data from your Apigee Account. You can look it up by going to your helloworld proxy.
+
+![image alt text](images/image_6.png)
+
+Positive
+: Note: Also change the port to 8080 if you are working on a Chromebook as the port 9001 cannot be used by docker containers.
+
+You can also have to change the name to match the one of your docker images. Change apigee/ci to apigee/ci-demo-walkthrough and also replace my-ci with a unique name that does not match another container running on your machine.
+
+![image alt text](images/image_7.png)
+
+A Description of the fields is within the file. After done with the modification run the script.
+
+```
+. ./setup/.run-image.sh
+```
+
+Script will output the docker container hash.
+
+### Access Jenkins
+
+Fire up a browser and hit http://localhost:8080 to access Jenkins UI.
+
+Your setup is successful if you see Jenkins with this picture (notice the color of the circles). Give it a couple of minutes to start and the grey circle to turn blue (except the "currency-v1-features")
+
+![image alt text](images/image_8.png)
+
+Please note that Jenkins will start all jobs upon start as it is configured to trigger each job every minute [why not git web hooks?](https://github.com/flogei/ci-demo-walkthrough/blob/master/docs/faq.md#why-not-git-web-hooks). Master and prod jobs should succeed but feature job will fail as we don't have a feature branch yet.
+
+Please follow [usage](https://github.com/flogei/ci-demo-walkthrough/blob/master/docs/usage.md) to see Jenkins in action.
```

Comment:
dead link

...also this should be required as I guess this codelab will replace the usage.md file?